content/contributing/documentation/content_guidelines.rst

```diff
@@ -238,27 +288,24 @@ folder ``doc_filename``.
 
 .. note::
    Previously, image filenames would mostly be named with numbers (e.g., :file:`feature01.png`) and
-   placed in a single ``media`` folder. While it is advised not to name your *new* images in that
-   fashion, it is also essential **not to rename unchanged files**, as doing this would double the
-   weight of renamed image files on the repository. They will eventually all be replaced as the
+   placed in a single :file:`media` folder. While it is advised not to name your *new* images in
+   that fashion, it is also essential **not to rename unchanged files**, as doing this would double
+   the weight of renamed image files on the repository. They will eventually all be replaced as the
    content referencing those images is updated.
 
-.. _contributing/alt-tags:
+.. _contributing/content/alt-tags:
 
 ALT tags
 --------
 
 An **ALT tag** is a *text alternative* to an image. This text is displayed if the browser fails to
 render the image. It is also helpful for users who are visually impaired. Finally, it helps
 search engines, such as Google, to understand what the image is about and index it correctly, which
-improves the :abbr:`SEO (Search Engine Optimization)` significantly.
+improves :abbr:`SEO (Search Engine Optimization)`.
 
 Good ALT tags are:
 
-- **Short** (one line maximum)
-- **Not a repetition** of a previous sentence or title
-- A **good description** of the action happening on the image
-- Easily **understandable** if read aloud
-
-.. seealso::
-   - :ref:`RST cheat sheet: image directive <contributing/image>`
+- **Short** (one line maximum);
+- **Not a repetition** of a previous sentence or title;
+- A **good description** of the action happening on the image;
+- Easily **understandable** if read aloud.
```

Comment:
Curious: 
- is it worth it to speak to and then [reference icons](https://www.odoo.com/documentation/master/contributing/documentation/rst_cheat_sheet.html#icons) here in this doc at all (maybe under images heading)?
- Also should we include a subsection titled **Spelling** to indicate that we generally prioritize American English vs. British English? This section *might* further be useful across the various language localizations like MX vs ES Spanish, FR vs BE French, etc.?